index.md

```diff
@@ -7,4 +7,32 @@ position: 1
 tags: overview, welcome
 ---
 
-This is a placeholder document for the index page.
\ No newline at end of file
+The {{site.tp}} Knowledge Base portal gives answers to common user inquiries or issues, identified by the development team in the form of How-To and Troubleshooting articles. The easiest way to find relevant information is to search for key words in the *Search Knowledge Base* box at the top right corner.
+
+If you don't find information about the exact issue/scenario here, continue with the next steps in order to utilize the full potential of our support service:
+
+# **Step 1:** Browse the support resources
+The {{site.ab}} support system provides a number of **[support resources](http://docs.telerik.com/platform/appbuilder/troubleshooting/support/support-resources)** which you can browse for solutions to your problem. You can apply the suggested workarounds and continue your work without submitting a support request.
```

Comment:
Should we use {{site.tap}} to refer to Telerik Platform or this is intended to be AB related only?  I'd like to add a link to the BS troubleshooting section too?